compose.static.yml

```diff
@@ -7,7 +7,7 @@ services:
       context: .
       dockerfile: testnet/Dockerfile
     ports:
-      - "3000:3000/tcp"
+      - "3000:3000/udp"
```

Comment:
I checked that nodes can connect to the bootstrap node with `ipv4/bootstrap/udp/3000/quic-v1` even though we set the protocol `/udp` here.